atom/browser/native_window_views.cc

```diff
@@ -613,6 +614,29 @@ void NativeWindowViews::SetProgressBar(double progress) {
 #endif
 }
 
+void NativeWindowViews::SetOverlayIcon(
+  const gfx::Image& overlay,
+  const std::string& description) {
```

Comment:
We prefer 4 spaces as indent when breaking a function declaration:
http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Function_Declarations_and_Definitions